### ISSUE TITLE
[compiler] warning on accessibility mistakes in markup

### ISSUE THREAD

**Issue by caridy** [salesforce/lwc#434]
Svelte and other frameworks are complaining about missing accessibility details. E.g.:

![image](https://user-images.githubusercontent.com/38969/41677333-f93ea9c0-7495-11e8-8f37-c410350fb2ad.png)

Aura on the other hand, uses a component test mechanism to warn you about potential issues. I think we should favor the compiler warning instead.
